finetune/model.py

```diff
@@ -71,10 +71,9 @@ def language_model_op(X, M, params, featurizer_state, mode, encoder):
     return lm_predict_op, language_model_state
 
 
-def masked_language_model_op(X, M, mlm_weights, mlm_ids, mlm_positions, params, featurizer_state, mode):
+def masked_language_model_op(X, mlm_weights, mlm_ids, mlm_positions, params, featurizer_state, mode):
```

Comment:
nice call with mlm_mask. unless I missed something, it seems like we also don't need mlm_weights either?

Reply:
Yeah I suppose that could be computed in graph with a scatter using mlm_positions?